package.json

```diff
@@ -5,9 +5,10 @@
     "action": "bash ./scripts/run_action.sh",
     "action:help": "npm run action",
     "action:list": "npm run action",
-    "clean": "rm -rf build www node_modules platforms/* plugins/*",
+    "clean": "rm -rf build node_modules www/* platforms/* plugins/*",
```

Comment:
I changed this to `www/*` as having `www` in the root is what makes this a "cordova" project, making it a little less annoying to run certain commands. 

I would also support reverting this and changing `platforms/* plugins/*` to `platforms plugins` instead

Reply:
Could you clarify what's annoying?

I'd prefer to remove all directories that are generated, so your second option. This way it's closer to a fresh clone.

Reply:
It's been a minute so I don't remember offhand 😅  I'll just switch to option #2